src/views/communities/Stories.tsx

```diff
@@ -133,7 +121,7 @@ export default function Stories() {
                     />
                 )}
                 {storiesCommunity.length > 0 &&
-                    storiesCommunity.map((s) => (
+                    storiesCommunity.slice(0, 5).map((s, index) => (
```

Comment:
please don't! I don't want beneficiaries to fetch 500+ (once we have that number) of stories and then slice it. Plus, it's extra wasted work for the server. Use pagination!